File: src/string/getInitials/test_get_initials.py
from get_initials import get_initials
import pytest


@pytest.mark.parametrize('input, expected',
                         [('Владимир Семенович Высоцкий', 'В.С.В.'),
                          ('Гуев Тимур Ахсарбекович', 'Г.Т.А.'),
                          ('Михаил Иванович Кузнецов', 'М.И.К.')])
def test_get_initials(input, expected):
    assert get_initials(input) == expected

File: src/string/getInitials/TestGetInitials.java
package string.getInitials;

import static org.junit.Assert.assertEquals;

import org.junit.jupiter.params.ParameterizedTest;
import org.junit.jupiter.params.provider.CsvSource;

public class TestGetInitials {
    @ParameterizedTest
    @CsvSource(delimiter = '|', value = { "Владимир Семенович Высоцкий|В.С.В.",
            "Гуев Тимур Ахсарбекович|Г.Т.А.",
            "Михаил Иванович Кузнецов|М.И.К." })
    void testGetInitials(String input, String expected) {
        assertEquals(expected, GetInitials.getInitials(input));
    }
}
